src/app/server/Server.cpp

```diff
@@ -332,6 +332,10 @@ CHIP_ERROR Server::Init(const ServerInitParams & initParams)
                                                                  &mCASESessionManager, mSubscriptionResumptionStorage);
     SuccessOrExit(err);
 
+#if CHIP_CONFIG_ENABLE_ICD_SERVER
+    chip::app::InteractionModelEngine::GetInstance()->SetICDManager(&mICDManager);
```

Comment:
```suggestion
    app::InteractionModelEngine::GetInstance()->SetICDManager(&mICDManager);
```

I know the line 2 above this has the extra `chip::` prefix too.  It shouldn't.